source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -233,8 +233,9 @@ export default class NetworkStatusStore extends Store {
       if (status)
         runInAction('assigning node status', () => {
           const { cardanoNodeID } = status;
+          const { isNodeSyncing } = this;
           this.cardanoNodeID = cardanoNodeID;
-          Object.assign(this, status);
+          Object.assign(this, status, { isNodeSyncing });
```

Comment:
@daniloprates I think this is not a proper fix. The problem appears to be that we do not update cardano status if we are not connected: https://github.com/input-output-hk/daedalus/blob/40ea71280e85f43d6565667a019513ee40c0a810/source/renderer/app/stores/NetworkStatusStore.js#L193
Let's chat about this once you start today...

Reply:
We need to be careful not to break something else. This caching was introduced by @DominikGuzei in the scope of this commit: https://github.com/input-output-hk/daedalus/commit/60d292f3f6e37e5acad3d619b2b10683e908a275

Reply:
Which was a part of this PR: https://github.com/input-output-hk/daedalus/pull/1178

Reply:
Perhaps we should cache this status only in test mode? @DominikGuzei @daniloprates

Reply:
Caching only in test mode would be a quick solution 👍  I don't think it brings much value for the user, only for app reloads which probably happen not that often

Reply:
@DominikGuzei let's discuss this today on our weekly call 🙏

Reply:
Done! @DominikGuzei @daniloprates